contracts/BondedSortitionPool.sol

```diff
@@ -54,6 +60,7 @@ contract BondedSortitionPool is AbstractSortitionPool {
             _minimumStake,
             _bondingContract,
             _minimumBondableValue,
+            _minimumBondableValue,
```

Comment:
Can't we leave `selectionBond` with the default value `0`?

Reply:
There should be no technical reason we couldn't.

Reply:
Sure!